frontend-react/src/components/header/ReportStreamHeader.tsx

```diff
@@ -62,6 +62,13 @@ export const ReportStreamHeader = () => {
                 >
                     <span>{FeatureName.DAILY_DATA}</span>
                 </USNavLink>
+                // <USNavLink
```

Comment:
Totally OK if intended for the future, just leave a note on it to indicate this.